src/components/SyntaxLookupWidget.res

```diff
@@ -87,6 +114,78 @@ let allItems = [
     category: Other,
     component: controlflow_ifelse,
   },
+  {
+    id: "integer-addition",
```

Comment:
this data is good for drafting and testing, but since we will include a ton of syntax constructs, I am pretty sure that this will not scale on a bundle level, so we need to factor everything out into some server side rendered solution of some sort.

So what I think would be best is to put the metadata in the frontmatter of each syntax construct file, e.g. `operators_integer_addition.mdx`:

```
---
id: "integer-addition"
keywords: ["plus", "add", "addition", "sum", "int", "integer"]
name: "+"
summary: "This is the `integer addition` operator."
category: "operators"
---
```

You can of course still hook your components up with the SyntaxLookupWidget to test your keywords, but we will remove that code pretty soon later on, just so you know.

Reply:
Thanks @ryyppy that makes sense, updated.